index.html

```diff
@@ -34,36 +42,78 @@
     <section class="ds-l-container ds-u-padding--1 ds-u-sm-padding--2 ds-u-md-padding--3 ds-u-lg-padding--4 ds-u-xl-padding--5 ds-u-color--white">
       <div class="ds-l-row">
         <div class="ds-l-col--12">
-          <div class="ds-u-font-size--h1 ds-u-color--white ds-u-font-weight--semibold"> How can ACOs benefit from BCDA? </div>
+          <div class="ds-u-font-size--h1 ds-u-color--white ds-u-font-weight--semibold"> How does BCDA benefit participants? </div>
         </div>
       </div>
       <div class="ds-l-row ds-u-margin-top--5">
         <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--4 benefit-prop">
-          <img src="/assets/img/homepage/ValueProp_1.svg" alt="Improve">
+          <img src="/assets/img/homepage/ValueProp_1.svg" width="50%" height="50%" alt="Improve">
```

Comment:
Should these icons be horizontally-centered now that they're smaller? they look a little awkward with left-alignment.

<img width="1108" alt="Screen Shot 2023-08-03 at 5 18 14 PM" src="https://github.com/CMSgov/bcda-static-site/assets/2308368/902f7bad-680b-4e8f-bb6d-08eace3f6202">

Reply:
the icons are still centered-left; should they be?

Reply:
They should be centered now!